nav2_dwb_controller/dwb_controller/src/dwb_controller.cpp

```diff
@@ -100,8 +100,10 @@ DwbController::on_deactivate(const rclcpp_lifecycle::State & state)
 {
   RCLCPP_INFO(get_logger(), "Deactivating");
 
+  action_server_->deactivate();
```

Comment:
Need to add `action_server_->activate()` to `DwbController::on_activate` for the server to be available next time the node goes active.

Reply:
good catch